server/controllers/chartsController.js

```diff
@@ -48,13 +49,26 @@ const postProcessData = (data, studyTotals) => {
   })
 
   processedData['N/A'] = notAvailableArray
-
+  Object.keys(processedData).forEach((key) => {
+    processedData[key].sort(function (studyA, studyB) {
+      return studyA.study === TOTALS_STUDY
+        ? -1
+        : studyB.study === TOTALS_STUDY
+        ? 1
+        : 0
+    })
```

Comment:
can we remove the nested ternary? Those are pretty tough to follow.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/137/commits/6c921e8f5dfccbd2c7666d80b951df5613819ba9